anomalib/pre_processing/pre_process.py

```diff
@@ -7,11 +7,111 @@
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import logging
 from typing import Optional, Tuple, Union
 
 import albumentations as A
 from albumentations.pytorch import ToTensorV2
 
+from anomalib.data.utils import get_image_height_and_width
+
+logger = logging.getLogger(__name__)
+
+
+def get_transforms(
```

Comment:
Should we also move this function to `data/utils`?

Reply:
i think this is still one of the most important components of `pre-processing` sub-package. If we move it to `data/utils`, then `pre_processing` would become quite a weak package that we could consider removing in the future

Reply:
In that case we could ask ourselves if we really need the `pre-processing` module

Reply:
yes, that's the question, I think

Reply:
I'm merging this one, we could discuss this later on